src/client/components/evaluate/ProjectEvaluateSidebar.tsx

```diff
@@ -103,14 +103,18 @@ const ProjectEvaluateSidebar = ({
     {
       key: "contiguity",
       name: "Contiguity",
-      status: true,
+      status:
+        geojson?.features.filter(f => f.properties.contiguity === "non-contiguous").length === 0
+          ? true
+          : false,
```

Comment:
In general, any code of the type `let my_var = some_boolean_expression ? true : false` can **always** be simplified to just `let my_var = some_boolean_expression`.

This is a similar code smell to things like:
```
if (some_boolean_expression) {
  return true;
} else {
  return false;
}
```

Which can always be simplified to simply: `return some_boolean_expression;`.


So in this specific case:
```suggestion
        geojson?.features.filter(f => f.properties.contiguity === "non-contiguous").length === 0,
```